docs/products/opensearch/howto/opensearch_alerting.rst

```diff
@@ -0,0 +1,120 @@
+-------------------
+OpenSearch alerting
+-------------------
+
+   OpenSearch alerting feature sends notifications when data from one or more indices meets certain conditions that can be customized.
+   Use case examples are such as monitoring for HTTP status code 503, CPU load average above certain percentage or watch for counts of a specific keyword in logs for a specific amount of interval,
+   notification to be configured to be sent via email, slack or webhooks and other destination.
+
+   In the following example, we are using a ``sample-host-health`` index as datasource to create a simple alert to check cpu load, action will be triggered when average of ``cpu_usage_percentage`` over ``3`` minutes is above ``75%``
```

Comment:
```suggestion
In the following example, we are using a ``sample-host-health`` index as datasource to create a simple alert to check cpu load, action will be triggered when average of ``cpu_usage_percentage`` over ``3`` minutes is above ``75%``
```

Reply:
we don't use horizontal line usually, and I don't think it is necessary in first paragraph